assets/javascripts/birdeck.js

```diff
@@ -21,9 +21,7 @@ function renderPost(post) {
 }
 
 function fetchPostsButton() {
-  $("#button-fetch-posts").on("click", function(){
-    fetchPosts()
-    })
+  $("#button-fetch-posts").on("click", fetchPosts)
```

Comment:
@stevekinney I misread! I thought you were erasing the entire `fetchPostsButton` function. I see now that you were referring on passing the function as a reference in the second argument of the `.on` function. I like this.